.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "edk2"]
```

Comment:
I have some doubts regarding this being a submodule.
If it's something needed just for the build phase, it can be installed then based on the wiki pages mentioned. Why do we need to keep its source in our repo?

Reply:
I wanted it to be continuously updated as we are doing with QEMU, since it is still under active development (mainly driven by requirements for KVM).